src/platform/plugins/shared/dashboard/public/dashboard_api/track_panel.ts

```diff
@@ -63,17 +61,12 @@ export function initializeTrackPanel(untilEmbeddableLoaded: (id: string) => Prom
 
       untilEmbeddableLoaded(id).then(() => {
         setScrollToPanelId(undefined);
-        if (scrollPosition) {
-          panelRef.ontransitionend = () => {
-            // Scroll to the last scroll position after the transition ends to ensure the panel is back in the right position before scrolling
-            // This is necessary because when an expanded panel collapses, it takes some time for the panel to return to its original position
-            window.scrollTo({ top: scrollPosition });
-            scrollPosition = undefined;
-            panelRef.ontransitionend = null;
-          };
-          return;
+        if (scrollPosition !== undefined) {
+          window.scrollTo({ top: scrollPosition });
+          scrollPosition = undefined;
+        } else {
+          panelRef.scrollIntoView({ block: 'start' });
```

Comment:
We don't need to wait for the transition anymore since the new layout engine doesn't have animations